hxd/fs/FileEntry.hx

```diff
@@ -1,34 +1,142 @@
 package hxd.fs;
 
-class FileEntry {
+/**
+	The base class representing a single file in the FileSystem.
 
+	Should not be instantiated directly, as specific implementation lies on the FileSystem type used.
+**/
+class FileEntry {
+	/**
+		The name of the file.
+	**/
 	public var name(default, null) : String;
+	/**
+		The local path of the file.

```

Comment:
The local path of the file from the resource root, for instance /myproject/res/ui/background/image.png will have a path of ui/background/image.png